*** DRILL - Exercises in Probability

1. Calculate the probability of flipping a balanced coin four times and getting each pattern: HTTH, HHHH and TTHH.

In [4]:
htth = (1/2) * (1/2) * (1/2) * (1/2)
hhhh = (1/2) * (1/2) * (1/2) * (1/2)
tthh = (1/2) * (1/2) * (1/2) * (1/2)
print(htth, hhhh, tthh)

0.0625 0.0625 0.0625


A: The probability is the same for all three since they each require a specific pattern. Each single step has same probability and they have the same amount of steps. 1/16  or  .0625 or 6.25%

2. If a list of people has 24 women and 21 men, then the probability of choosing a man from the list is 21/45. What is the probability of not choosing a man?

In [6]:
not_a_man_prob = 24/45
print(not_a_man_prob)

0.5333333333333333


A: 0.5333333333333333 or 53.333%

3. The probability that Bernice will travel by plane sometime in the next year is 10%. The probability of a plane crash at any time is .005%. What is the probability that Bernice will be in a plane crash sometime in the next year?

In [7]:
bernice_die_by_plane = .1 * .00005
print(bernice_die_by_plane)

5e-06


5e-06 or .000005 or .0005% chance that Bernice will have a very unpleasant place ride next year

4. A data scientist wants to study the behavior of users on the company website. Each time a user clicks on a link on the website, there is a 5% chance that the user will be asked to complete a short survey about their behavior on the website. The data scientist uses the survey data to conclude that, on average, users spend 15 minutes surfing the company website before moving on to other things. What is wrong with this conclusion?

A: I see a couple of problems with this. First, the chance is on click and that will cause the survey to go to people who are clicking the most things and on the site longer than most. It does not mention whether the time spent includes or does not include the survey time, if left unclear in the survey could have viewers giving mixed numbers.